### PR TITLE
Add openstack terminal in infra view

### PR DESCRIPTION
**Pull request by snigle**

### CODE REVIEW

File: client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.controller.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        this.load();
+        this.savePrefs();
+    }
+
+    load () {
+        if (this.minimized) {
+            return;
+        }
+
+        if (!this.session.loading && (this.session.hasErrors || _.isEmpty(this.session.data))) {
```
Comment (Mboulianne) [inline client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.controller.js]: this IF is useless cache does the same job for us no?

File: client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="terminal col-md-offset-2 col-sm-offset-3"
+     data-ng-class="{ maximized : OpenstackClientCtl.maximized, minimized : OpenstackClientCtl.minimized }"
+     >
+        <div class="header" data-ng-click="OpenstackClientCtl.clickBar()">
+            <div class="title" data-translate="cpci_openstack_client">
+            </div>
+            <div class="buttons">
+                <button class="no-style" type="button"
+                 data-ng-if="!OpenstackClientCtl.minimized"
+                 data-ng-click="OpenstackClientCtl.minimize();$event.stopPropagation()"><i class="fa fa-window-minimize"></i></button>
```
Comment (Mboulianne) [inline client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html]: you can pass the $event to minimize and call stopPropagation in the controller.

File: client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        </div>
+        <div class="settings">
+            <label data-translate="cpci_openstack_client_regions"></label>
+            <select ng-if="!OpenstackClientCtl.regions.loading"
+                class="form-control"
+                ng-change="OpenstackClientCtl.Service.setRegion(OpenstackClientCtl.region)"
+                ng-model="OpenstackClientCtl.region"
+                >
+                    <option value="{{OpenstackClientCtl.emptyOption}}" hidden data-translate="cpci_openstack_client_select_region"></option>
+                    <option ng-repeat="region in OpenstackClientCtl.regions.data track by region" value="{{region}}">
```
Comment (Mboulianne) [inline client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html]: select ng-options should cover that for you... but I remember you had issue with this T_T

File: client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<div id="openstackClient" data-ng-controller="CloudProjectComputeInfrastructureOpenstackClientCtrl as OpenstackClientCtl" >
+    <div class="margin"></div>
+    <div class="terminal col-md-offset-2 col-sm-offset-3"
+     data-ng-class="{ maximized : OpenstackClientCtl.maximized, minimized : OpenstackClientCtl.minimized }"
```
Comment (Mboulianne) [inline client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html]: indentation

File: client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html (hunk 3 above)
Comment (snigle) [reply client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html]: yes I find out the bug but I need the first hidden option

**Comment by Netnix** [comment]
Already merged using another branch.

**Pull request closed without merging**